src/main/java/ui/MoveMethodTableModel.java

```diff
@@ -126,8 +132,40 @@ public int getRowCount() {
 
     @Override
     public void setValueAt(Object value, int virtualRow, int columnIndex) {
-        isSelected[virtualRows.get(virtualRow)] = (Boolean) value;
+        final int rowIndex = virtualRows.get(virtualRow);
+        final boolean select = (Boolean) value;
+        setValueAtRowIndex(select, rowIndex, true);
+
         fireTableCellUpdated(virtualRow, columnIndex);
+        fireTableDataChanged();
+    }
+
+    private void setValueAtRowIndex(boolean select, int rowIndex, boolean forceSelectInConflicts) {
+        if (!isActive[rowIndex]) {
+            return;
+        }
+
+        for (int i = 0; i < refactorings.size(); i++) {
```

Comment:
Now, it is hard to understand why we need it here. I suggest to extract code from 148 to 166 lines to a separate method and give it a clear name.

Reply:
Done.